src/oidc/index.ts

```diff
@@ -192,12 +193,14 @@ router.post(
                     },
                     gtm: GTM,
                 });
-            }
-            else {
-                const privacy = AccountService.updatePrivacy(
```

Comment:
@JustinaMary Returning the model into this `private` const makes no sense to me. I would have expected `await privacy.save()` after this line. 

The code block now:
1. Gets the account object for the sub thats found for the authenticationToken
2. Adjusts the properties on the account object
3. Saves the account object (take note that these are async requests)